ModelInstaller.iss

```diff
@@ -0,0 +1,177 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "ModelViewer"
+#define MyAppVersion "1.0"
+#define MyAppPublisher "ModelView team"
+#define MyAppURL "http://www.example.com/"
+#define MyAppExeName "ModelViewerRun.exe"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{177062CE-160C-445B-AF44-87C0DB0F72CC}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+DefaultDirName={autopf}\{#MyAppName}
+DisableProgramGroupPage=yes
+; Uncomment the following line to run in non administrative install mode (install for current user only.)
+;PrivilegesRequired=lowest
+OutputBaseFilename=mysetup
+Compression=lzma
+SolidCompression=yes
+WizardStyle=modern
+
+[Languages]
+Name: "english"; MessagesFile: "compiler:Default.isl"
+
+[Tasks]
+Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"; Flags: unchecked
+
+[Files]
+Source: "C:\Users\Zunyue\Desktop\cs561_project\ModelViewer\ModelViewer\x64\Release\ModelViewerRun.exe"; DestDir: "{app}"; Flags: ignoreversion

```

Comment:
I think this is fine for now, but it will only work on your computer.  We should make a future story for adding variables like:
`#define MyAppFolder "C:\Users\Zunyue\Desktop\cs561_project\ModelViewer\

#define MyQtFolder "C:\Qt\5.15.1\msvc2019_64\"`

Reply:
Disregard my previous comments.  I pulled your branch and used the script to make an installer!